source/administration/version-archive.rst

```diff
@@ -1,27 +1,35 @@
 Version Archive
 ===========================
 
+To verify the GPG signature of a Mattermost release, use the public key stored in Keybase: https://keybase.io/mattermost.
```

Comment:
@crspeller Added a link to keybase at top of the base.

Reply:
Looks good.